boards/shields/wiznet_wiz550io/wiznet_wiz550io.overlay

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2024 Grant Ramsay
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
```

Comment:
It looks like the Arduino intermediate board that converts the connector on this board to the Arduino pinout is discontinued. Not sure if we care about the inability to buy a required piece of this or not.

In general though, since this shield isn't using the Arduino-style connector (e.g. pin locations and physical dimensions of the connectors), should this be binding directly to the arduino_spi alias?